.eslintignore

```diff
@@ -5,4 +5,5 @@
 **/packages/migration/src/module.js
 bin/build/*
 build/*
+docs/*
```

Comment:
Reason?

Note that we can lint (JS) code samples in Markdown files with ESLint, so this change prevents doing that.

Reply:
I see, ESlint was expecting `@web-stories-wp/i18n` to be added as a dependency in root `package.json`

Reply:
Because it is used in a code snippet https://github.com/google/web-stories-wp/blob/main/docs/external-template-creation.md

Reply:
Will suppress the error instead in doc.

Reply:
A smarter way to do that would be to disable the `import/no-extraneous-dependencies` rule for `docs/` here:

https://github.com/google/web-stories-wp/blob/8aaa7b96fa1a1e22bffc3ef06f654f9881fc663d/.eslintrc#L446-L463